e2e_test/source/basic/inlcude_key_as.slt

```diff
@@ -48,9 +48,25 @@ WITH (
 	topic = 'upsert_json')
 FORMAT PLAIN ENCODE JSON
 
+statement ok
+create table additional_columns (a int)
+include key as key_col
+include partition as partition_col
+include offset as offset_col
+include timestamp as timestamp_col
+include header as header_col
+WITH (
+	connector = 'kafka',
+    properties.bootstrap.server = 'message_queue:29092',
+	topic = 'kafka_additional_columns')
+FORMAT PLAIN ENCODE JSON
+
 statement ok
 select * from upsert_students_default_key;
 
+statement ok
+select * from additional_columns;
```

Comment:
return a result here?

Reply:
it works as a trigger to ingest data, we check the results below. 
I follow the practice used in `e2e_test/source/basic/kafka.slt`